test/integration-no-mock/config-management/app-config-create.js

```diff
@@ -14,6 +14,7 @@
  */
 
 const async = require('async');
+const cloneDeep = require('lodash.clonedeep');
```

Comment:
I might be missing something, but this does not appear to be used.

Reply:
Fixed